packages/ckeditor5-dev-tests/bin/testautomated.js

```diff
@@ -12,11 +12,7 @@ import * as tests from '../lib/index.js';
 const options = tests.parseArguments( process.argv.slice( 2 ) );
 
 if ( options.files.length === 0 ) {
-	if ( options.cwd.endsWith( 'ckeditor5' ) ) {
-		options.files = [ '*' ];
-	} else {
-		options.files = [ '/' ];
-	}
+	options.files = [ '*' ];
```

Comment:
This is needed to be able to run `pnpm run test` without any parameters from a commercial repository and still run all tests.